### PR TITLE
Add bootstrap-avatars  w/ npm auto-update

### PR DESCRIPTION
**Pull request by bwsky-a**
Adding bootstrap-avatars package

### CODE REVIEW

File: packages/b/bootstrap-avatars.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "bootstrap-avatars",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*"
+        ]
```
Comment (MattIPv4) [inline packages/b/bootstrap-avatars.json:25]: 👋 This package does not include a `dist` directory: [glob-tool](https://www.digitalocean.com/community/tools/glob#?comments=true&glob=&matches=false&tests=%2F%2F%20Imported%20from%20bootstrap-avatars%20NPM%20package&tests=css%2Fmdb.dark.min.css&tests=css%2Fmdb.dark.rtl.min.css&tests=css%2Fmdb.min.css&tests=css%2Fmdb.rtl.min.css&tests=basic.html&tests=carousel.html&tests=index.html&tests=inside-navbar.html&tests=profile.html&tests=shadow.html&tests=square.html&tests=testimonials.html&tests=with-content.html&tests=img%2Fmdb-favicon.ico&tests=src%2Fjs%2Fbootstrap%2Fdist%2Falert.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Falert.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Falert.js&tests=src%2Fjs%2Ffree%2Falert.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Fbackdrop.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Fbackdrop.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fbase-component.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fbase-component.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fbase-component.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fbutton.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fbutton.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fbutton.js&tests=src%2Fjs%2Ffree%2Fbutton.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fcarousel.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fcarousel.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fcarousel.js&tests=src%2Fjs%2Ffree%2Fcarousel.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Flib%2Fclass-names.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fhandlers%2Fclick-rail.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fcollapse.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fcollapse.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fcollapse.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Fcomponent-functions.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Fcomponent-functions.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Flib%2Fcss.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fdata.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fdom%2Fdata.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fdom%2Fdata.js&tests=src%2Fjs%2Fmdb%2Fdom%2Fdata.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Flib%2Fdom.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fhandlers%2Fdrag-thumb.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdropdown.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fdropdown.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fdropdown.js&tests=src%2Fjs%2Ffree%2Fdropdown.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fevent-handler.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fdom%2Fevent-handler.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fdom%2Fevent-handler.js&tests=src%2Fjs%2Fmdb%2Fdom%2Fevent-handler.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Flib%2Fevent-manager.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Ffocustrap.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Ffocustrap.js&tests=src%2Fjs%2Fmdb%2Futil%2FfocusTrap.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Findex.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Findex.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Findex.js&tests=src%2Fjs%2Fmdb%2Futil%2Findex.js&tests=src%2Fjs%2Fmdb%2Futil%2Ftouch%2Findex.js&tests=src%2Fjs%2Ffree%2Finput.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fhandlers%2Fkeyboard.js&tests=src%2Fjs%2Fmdb%2Futil%2Fkeycodes.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fmanipulator.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fdom%2Fmanipulator.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fdom%2Fmanipulator.js&tests=src%2Fjs%2Fmdb%2Fdom%2Fmanipulator.js&tests=src%2Fjs%2Fmdb.free.js&tests=js%2Fmdb.min.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fmodal.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fmodal.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fmodal.js&tests=src%2Fjs%2Ffree%2Fmodal.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fhandlers%2Fmouse-wheel.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Foffcanvas.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Foffcanvas.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Foffcanvas.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fpopover.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fpopover.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fpopover.js&tests=src%2Fjs%2Ffree%2Fpopover.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fprocess-scroll-diff.js&tests=src%2Fjs%2Ffree%2Frange.js&tests=src%2Fjs%2Ffree%2Fripple.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Fsanitizer.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Fsanitizer.js&tests=src%2Fjs%2Fmdb%2Futil%2Fsanitizer.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Futil%2Fscrollbar.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Futil%2Fscrollbar.js&tests=src%2Fjs%2Fmdb%2Futil%2Fscrollbar.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fscrollspy.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fscrollspy.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fscrollspy.js&tests=src%2Fjs%2Ffree%2Fscrollspy.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fselector-engine.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Fdom%2Fselector-engine.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Fdom%2Fselector-engine.js&tests=src%2Fjs%2Fmdb%2Fdom%2Fselector-engine.js&tests=src%2Fjs%2Fmdb%2Futil%2Fstack.js&tests=src%2Fjs%2Fmdb%2Futil%2Ftouch%2Fswipe.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftab.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Ftab.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Ftab.js&tests=src%2Fjs%2Ffree%2Ftab.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftoast.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Ftoast.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Ftoast.js&tests=src%2Fjs%2Ffree%2Ftoast.js&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftooltip.js&tests=src%2Fjs%2Fbootstrap%2Fmdb-prefix%2Ftooltip.js&tests=src%2Fjs%2Fbootstrap%2Fsrc%2Ftooltip.js&tests=src%2Fjs%2Ffree%2Ftooltip.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fhandlers%2Ftouch.js&tests=src%2Fjs%2Fmdb%2Futil%2Ftouch%2FtouchUtil.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Fupdate-geometry.js&tests=src%2Fjs%2Fmdb%2Fperfect-scrollbar%2Flib%2Futil.js&tests=package.json&tests=src%2Fjs%2Fbootstrap%2Fdist%2Falert.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fbase-component.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fbutton.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fcarousel.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fcollapse.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fdata.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdropdown.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fevent-handler.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fmanipulator.js.map&tests=css%2Fmdb.dark.min.css.map&tests=css%2Fmdb.dark.rtl.min.css.map&tests=css%2Fmdb.min.css.map&tests=js%2Fmdb.min.js.map&tests=css%2Fmdb.rtl.min.css.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fmodal.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Foffcanvas.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fpopover.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fscrollspy.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Fdom%2Fselector-engine.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftab.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftoast.js.map&tests=src%2Fjs%2Fbootstrap%2Fdist%2Ftooltip.js.map&tests=README.md&tests=License.pdf&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_accordion.scss&tests=src%2Fscss%2Fbootstrap%2F_accordion.scss&tests=src%2Fscss%2Ffree%2F_accordion.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_alert.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_alert.scss&tests=src%2Fscss%2Fbootstrap%2F_alert.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_alert.scss&tests=src%2Fscss%2Ffree%2F_alert.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Futilities%2F_api.scss&tests=src%2Fscss%2Fbootstrap%2Futilities%2F_api.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_backdrop.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_backdrop.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_badge.scss&tests=src%2Fscss%2Fbootstrap%2F_badge.scss&tests=src%2Fscss%2Ffree%2F_badge.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_border-radius.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_border-radius.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_box-shadow.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_box-shadow.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_breadcrumb.scss&tests=src%2Fscss%2Fbootstrap%2F_breadcrumb.scss&tests=src%2Fscss%2Ffree%2F_breadcrumb.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_breakpoints.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_breakpoints.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_button-group.scss&tests=src%2Fscss%2Fbootstrap%2F_button-group.scss&tests=src%2Fscss%2Ffree%2F_button-group.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_buttons.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_buttons.scss&tests=src%2Fscss%2Fbootstrap%2F_buttons.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_buttons.scss&tests=src%2Fscss%2Ffree%2F_buttons.scss&tests=src%2Fscss%2Ffree%2Fmixins%2F_buttons.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_card.scss&tests=src%2Fscss%2Fbootstrap%2F_card.scss&tests=src%2Fscss%2Ffree%2F_card.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_caret.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_caret.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_carousel.scss&tests=src%2Fscss%2Fbootstrap%2F_carousel.scss&tests=src%2Fscss%2Ffree%2F_carousel.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_clearfix.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_clearfix.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_clearfix.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_clearfix.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_close.scss&tests=src%2Fscss%2Fbootstrap%2F_close.scss&tests=src%2Fscss%2Ffree%2F_close.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_color-scheme.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_color-scheme.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_colored-links.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_colored-links.scss&tests=src%2Fscss%2Ffree%2F_colors.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_container.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_container.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_containers.scss&tests=src%2Fscss%2Fbootstrap%2F_containers.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_deprecate.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_deprecate.scss&tests=src%2Fscss%2Ffree%2F_deprecated.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_dropdown.scss&tests=src%2Fscss%2Fbootstrap%2F_dropdown.scss&tests=src%2Fscss%2Ffree%2F_dropdown.scss&tests=src%2Fscss%2Ffree%2F_flag.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_floating-labels.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_floating-labels.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_form-check.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_form-check.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_form-check.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_form-control.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_form-control.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_form-control.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_form-file.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_form-range.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_form-range.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_form-range.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_form-select.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_form-select.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_form-select.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_form-text.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_form-text.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_forms.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_forms.scss&tests=src%2Fscss%2Fbootstrap%2F_forms.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_forms.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_functions.scss&tests=src%2Fscss%2Fbootstrap%2F_functions.scss&tests=src%2Fscss%2Ffree%2F_functions.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_gradients.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_gradients.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_grid.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_grid.scss&tests=src%2Fscss%2Fbootstrap%2F_grid.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_grid.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_helpers.scss&tests=src%2Fscss%2Fbootstrap%2F_helpers.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_image.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_image.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_images.scss&tests=src%2Fscss%2Fbootstrap%2F_images.scss&tests=src%2Fscss%2Ffree%2F_images.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_input-group.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_input-group.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_input-group.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_labels.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_labels.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_list-group.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_list-group.scss&tests=src%2Fscss%2Fbootstrap%2F_list-group.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_list-group.scss&tests=src%2Fscss%2Ffree%2F_list-group.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_lists.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_lists.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_mixins.scss&tests=src%2Fscss%2Fbootstrap%2F_mixins.scss&tests=src%2Fscss%2Ffree%2F_mixins.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_modal.scss&tests=src%2Fscss%2Fbootstrap%2F_modal.scss&tests=src%2Fscss%2Ffree%2F_modal.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_nav.scss&tests=src%2Fscss%2Fbootstrap%2F_nav.scss&tests=src%2Fscss%2Ffree%2F_nav.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_navbar.scss&tests=src%2Fscss%2Fbootstrap%2F_navbar.scss&tests=src%2Fscss%2Ffree%2F_navbar.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_offcanvas.scss&tests=src%2Fscss%2Fbootstrap%2F_offcanvas.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_pagination.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_pagination.scss&tests=src%2Fscss%2Fbootstrap%2F_pagination.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_pagination.scss&tests=src%2Fscss%2Ffree%2F_pagination.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_placeholders.scss&tests=src%2Fscss%2Fbootstrap%2F_placeholders.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_popover.scss&tests=src%2Fscss%2Fbootstrap%2F_popover.scss&tests=src%2Fscss%2Ffree%2F_popover.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_position.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_position.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_progress.scss&tests=src%2Fscss%2Fbootstrap%2F_progress.scss&tests=src%2Fscss%2Ffree%2F_progress.scss&tests=src%2Fscss%2Ffree%2F_range.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_ratio.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_ratio.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_reboot.scss&tests=src%2Fscss%2Fbootstrap%2F_reboot.scss&tests=src%2Fscss%2Ffree%2F_reboot.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_reset-text.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_reset-text.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_resize.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_resize.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fvendor%2F_rfs.scss&tests=src%2Fscss%2Fbootstrap%2Fvendor%2F_rfs.scss&tests=src%2Fscss%2Ffree%2F_ripple.scss&tests=src%2Fscss%2Ffree%2Fmixins%2F_ripple.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_root.scss&tests=src%2Fscss%2Fbootstrap%2F_root.scss&tests=src%2Fscss%2Ffree%2F_root.scss&tests=src%2Fscss%2Ffree%2F_scrollspy.scss&tests=src%2Fscss%2Ffree%2F_shadows.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_spinners.scss&tests=src%2Fscss%2Fbootstrap%2F_spinners.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_stacks.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_stacks.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_stretched-link.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_stretched-link.scss&tests=src%2Fscss%2Fcustom%2F_styles.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_table-variants.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_table-variants.scss&tests=src%2Fscss%2Ffree%2Fmixins%2F_table-variants.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_tables.scss&tests=src%2Fscss%2Fbootstrap%2F_tables.scss&tests=src%2Fscss%2Ffree%2F_tables.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_text-truncate.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_text-truncate.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_text-truncation.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_text-truncation.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_toasts.scss&tests=src%2Fscss%2Fbootstrap%2F_toasts.scss&tests=src%2Fscss%2Ffree%2F_toasts.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_tooltip.scss&tests=src%2Fscss%2Fbootstrap%2F_tooltip.scss&tests=src%2Fscss%2Ffree%2F_tooltip.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_transition.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_transition.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_transitions.scss&tests=src%2Fscss%2Fbootstrap%2F_transitions.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_type.scss&tests=src%2Fscss%2Fbootstrap%2F_type.scss&tests=src%2Fscss%2Ffree%2F_type.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_utilities.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_utilities.scss&tests=src%2Fscss%2Fbootstrap%2F_utilities.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_utilities.scss&tests=src%2Fscss%2Ffree%2F_utilities.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fforms%2F_validation.scss&tests=src%2Fscss%2Fbootstrap%2Fforms%2F_validation.scss&tests=src%2Fscss%2Ffree%2Fforms%2F_validation.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2F_variables.scss&tests=src%2Fscss%2Fbootstrap%2F_variables.scss&tests=src%2Fscss%2Fcustom%2F_variables.scss&tests=src%2Fscss%2Ffree%2F_variables.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_visually-hidden.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fmixins%2F_visually-hidden.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_visually-hidden.scss&tests=src%2Fscss%2Fbootstrap%2Fmixins%2F_visually-hidden.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fhelpers%2F_vr.scss&tests=src%2Fscss%2Fbootstrap%2Fhelpers%2F_vr.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fbootstrap-grid.scss&tests=src%2Fscss%2Fbootstrap%2Fbootstrap-grid.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fbootstrap-reboot.scss&tests=src%2Fscss%2Fbootstrap%2Fbootstrap-reboot.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fbootstrap-utilities.scss&tests=src%2Fscss%2Fbootstrap%2Fbootstrap-utilities.scss&tests=src%2Fscss%2Fbootstrap-rtl-fix%2Fbootstrap.scss&tests=src%2Fscss%2Fbootstrap%2Fbootstrap.scss&tests=src%2Fscss%2Fmdb.free.scss)

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/b/bootstrap-avatars.json (hunk 1 above)
Comment (bwsky-a) [reply packages/b/bootstrap-avatars.json]: Hi, sorry for the oversight, I've already made some corrections

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
👋 It looks like your latest commit (other than my merge) looks to have introduced some additional changes (which I suspect are from case-insensitive naming collisions) -- would you mind reverting these please?

**Comment by bwsky-a** [comment]
Hi, I have already reverted my previous changes. Hope everything is fine now :)

File: packages/b/bootstrap-avatars.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "bootstrap-avatars",
+  "filename": "js/mdb.min.js",
```
Comment (MattIPv4) [inline packages/b/bootstrap-avatars.json:3]:
```suggestion
  "filename": "bootstrap-avatars.min.css",
```

**Review by MattIPv4 — APPROVED** [review]
LGTM, ty!

**Pull request merged**